packages/alpine/files/initramfs-init

```diff
@@ -375,16 +383,16 @@ if grep -q netboot /proc/cmdline; then
   sync
   # Create mountpoints
   ebegin "Create mountpoints"
-  mkdir -p /media/root-rw /run/rootfsbase $sysroot/media/root-rw $sysroot/run/rootfsbase
+  mkdir -p $rootRW $rootfsbase $sysroot$rootRW $sysroot$rootfsbase
```

Comment:
if I understand correctly this:

```
/media/root-rw /run/rootfsbase $sysroot/media/root-rw $sysroot/run/rootfsbase
```

Becomes this

```
/run/overlay /run/rootfsbase $sysroot/run/overlay $sysroot/run/rootfsbase
```

so that means that `/media/root-rw` got changed to `/run/overlay` and same to the one that is prefixed by the `$sysroot`

why did the location change?

Reply:
to have it in line with what immucore does. We mount a writable /run/overlay and then submounts things there (/etc /var and so on, basically the ephemeral parts)

So in this case, this initrd was just doing its own thing. In this case it should not matter much as nothing should be using that, especially during live boot, but having ti be the same makes it easier to understand both what dracut does and what we do :)